src/core/Track.cpp

```diff
@@ -637,4 +637,19 @@ BoolModel *Track::getMutedModel()
 	return &m_mutedModel;
 }
 
+void Track::setName( const QString & newName )
```

Comment:
```suggestion
void Track::setName(const QString& newName)
```
One more style suggestion

Reply:
Done with commit ffb2a669.